eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -204,40 +204,24 @@ jobs:
               Write-Host "##vso[task.setvariable variable=AZURE_SDK_VERSION]$version"
             displayName: Create package info JSON file
 
-          # Building from the root of the repo at this point because of the
-          # nature of the CMake build system. The SoureDirectory should probably
-          # change.
-          - task: PowerShell@2
-            inputs:
-              targetType: filePath
-              filePath: eng/scripts/New-ReleaseArtifact.ps1
-              arguments: >-
-                -SourceDirectory ./
-                -PackageSpecPath $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/package-info.json
-                -Destination $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/vcpkg
-                -Workspace $(Agent.TempDirectory)/vcpkg-${{ artifact.Name }}
-              pwsh: true
-            displayName: Compress publish artifact and stage for publishing
-
           - task: Powershell@2
             inputs:
               targetType: inline
               script: |
                 # If there is no vcpkg directory skip staging the vcpkg port artifacts
-                if (!(Test-Path sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Name }}/vcpkg/)) {
+                if (!(Test-Path build/vcpkg/ports/${{ artifact.VcpkgPortName }})) {
                   exit
```

Comment:
Add an output message to console stating that the `build/vcpkg/ports/${{ artifact.VcpkgPortName }}` wasn't found. This will help debugging.